src/command_modules/azure-cli-network/azure/cli/command_modules/network/custom.py

```diff
@@ -338,26 +346,184 @@ def set_lb_rule(
         load_distribution='default', floating_ip=None, idle_timeout=None):
     ncf = _network_client_factory()
     lb = ncf.load_balancers.get(resource_group_name, load_balancer_name)
-    item = _get_lb_property(lb.load_balancing_rules, item_name)
+    item = _get_property(lb.load_balancing_rules, item_name)
+
     _set_param(item, 'protocol', protocol)
     _set_param(item, 'frontend_port', frontend_port)
     _set_param(item, 'backend_port', backend_port)
     _set_param(item, 'idle_timeout_in_minutes', idle_timeout)
     _set_param(item, 'load_distribution', load_distribution)
+
     if frontend_ip_name is not None:
         item.frontend_ip_configuration = \
-            _get_lb_property(lb.frontend_ip_configurations, frontend_ip_name)
+            _get_property(lb.frontend_ip_configurations, frontend_ip_name)
+
     if floating_ip is not None:
         item.enable_floating_ip = floating_ip == 'true'
+
     if backend_address_pool_name is not None:
         item.backend_address_pool = \
-            _get_lb_property(lb.backend_address_pools, backend_address_pool_name)
-    if probe_name == "":
+            _get_property(lb.backend_address_pools, backend_address_pool_name)
+
+    if probe_name == '':
         item.probe = None
     elif probe_name is not None:
-        item.probe = _get_lb_property(lb.probes, probe_name)
+        item.probe = _get_property(lb.probes, probe_name)
+
     return ncf.load_balancers.create_or_update(resource_group_name, load_balancer_name, lb)
 
+# NIC commands
+
+def set_nic(resource_group_name, network_interface_name, network_security_group=None,
+            enable_ip_forwarding=None, internal_dns_name_label=None):
```

Comment:
In short, no. Because it is an update command, you have to specify true or false (to differentiate between None, i.e. leave unchanged). The argparse choices field only accepts strings, not bool values. 

---
In reply to: [73234138](https://github.com/Azure/azure-cli/pull/592#discussion_r73234138) [](ancestors = 73234138)